templates/search/blocks/search_filters.html

```diff
@@ -8,7 +8,7 @@ <h2 class="search-sort-view__heading">
         {{ form.sort_by.errors }}
         {{ form.sort_by }}
 
-        {{ 'sort_by'|include_hidden_fields:form }}
+        {{ 'catalogue-sort-form-mobile'|include_hidden_fields:form }}
```

Comment:
This feels a bit wrong. How do we know what magic value to use here? It's at least semi-clear when the field name is used.

Reply:
The doc strings in the method should be helpful.

Reply:
Something like `{% render_form_fields_as_hidden form=form exclude='sort_by' %}` would be far better!

Reply:
same as 

> @ababic how will this cover [here-include_hidden_fields](https://github.com/nationalarchives/ds-wagtail/blob/f17e917ada03a73c8a6062a5abaadf9156cf579d/etna/search/templatetags/search_tags.py#L93-L95) and [here-test_render_invalid_date_range_all_dates](https://github.com/nationalarchives/ds-wagtail/blob/f17e917ada03a73c8a6062a5abaadf9156cf579d/etna/search/tests/test_views.py#L641-L643)